packages/amplify-appsync-simulator/src/server/operations.ts

```diff
@@ -33,54 +30,29 @@ export class OperationServer {
     private subscriptionServer: SubscriptionServer,
   ) {
     this.port = config.port;
-    this.app = express();
-    this.app.use(express.json({ limit: MAX_BODY_SIZE }));
-    this.app.use(cors());
-    this.app.post('/graphql', this.handleRequest.bind(this));
-    this.app.get('/api-config', this.handleAPIInfoRequest.bind(this));
-    this.app.use('/', express.static(STATIC_ROOT));
+    this._app = express();
+    this._app.use(express.json({ limit: MAX_BODY_SIZE }));
+    this._app.use(cors());
+    this._app.post('/graphql', this.handleRequest.bind(this));
+    this._app.get('/api-config', this.handleAPIInfoRequest.bind(this));
+    this._app.use('/', express.static(STATIC_ROOT));
     this.server = null;
   }
 
-  async start() {
-    if (this.server) {
-      throw new Error('Server is already running');
-    }
-
-    if (!this.port) {
-      this.port = await portfinder.getPortPromise({
-        startPort: BASE_PORT,
-        stopPort: MAX_PORT,
-      });
-    }
-
-    this.server = this.app.listen(this.port);
+  async start() {}
```

Comment:
why do we need these empty methods if this class is not an interface implementation of some generic server?